src/logger.rs

```diff
@@ -112,33 +113,47 @@ pub fn get_derivability_results<L: SynthLanguage>(
     ruleset: &Ruleset<L>,
     derive_type: DeriveType,
     baseline: &Ruleset<L>,
+    derive: bool,
 ) -> ((usize, usize), (Duration, Duration), Value) {
-    let limits = if let DeriveType::AllRules = derive_type {
-        Limits {
-            iter: 2,
-            node: 100_000,
-        }
+    if !derive {
```

Comment:
I think it's weird/confusing to have this check in this function. it would make more sense to have it in `write_output`. Like, we shouldn't be calling a function called `get_derivability_results` if we don't want to actually run derivability

Reply:
it's a lot easier to write this way

Reply:
but i'll change it